Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,44 @@
+public class View: Viewable {
+	var ui: Receivable 
+	var guess = ""
```

Comment:
We discussed a preferred coding style for handling optional properties, and that blank variables are a code smell
```swift
public var foo: Type?
```
or
```swift
public var foo: Type!
```